contracts/interfaces/IDistributable.sol

```diff
@@ -19,4 +21,17 @@ interface IDistributable {
      * @param distributor The address of the new distributor
      */
     function setDistributor(address distributor) external;
```

Comment:
Maybe switch to `transfer` instead of `set` for consistency

Reply:
`transferOwnership` => `transferDistributorship` sounds weird we could maybe do `transferDistributor` but that is grammatically incorrect so idk